evap/evaluation/models.py

```diff
@@ -1547,7 +1544,7 @@ class UserProfile(AbstractBaseUser, PermissionsMixin):
     is_active = models.BooleanField(default=True, verbose_name=_("active"))
 
     class Meta:
-        ordering = ["last_name", "first_name", "email"]
+        ordering = [Lower(Concat("last_name", "first_name", "email"))]
```

Comment:
sorry, we only now noticed that we may not want `Concat` afterall; it should just sort for the `Lower` of each individually